apps/web/lib/hooks/useSuperfetch.ts

```diff
@@ -35,9 +40,31 @@ export default function useSuperfetch() {
         if (res.ok) {
           const data = await res.json().catch(() => ({}));
           return { ...data, httpStatus: res.status };
+        } else if (res.status === 400) {
+          const hardcodedError = "10.10";
+          showToast({
+            variant: "error",
+            message:
+              t(dict.title) + hardcodedError + " - " + t(dict[hardcodedError]),
+          });
+        } else if (res.status !== 200 || 400 || 401) {
```

Comment:
I am a bit confused about what this line does 👀 you are not really comparing anything here, since this will always be truthy (`400` is always truthy). Maybe you can simplify it like below. You already check for 200 and 201 in the first if `(res.ok)`, then for 400 in the else if condition. 

```suggestion
        } else if (res.status > 401) {
```

Reply:
I changed it like you suggested but I have to test it more when BE will be ready.